agent/command/expansion_test.go

```diff
@@ -63,7 +63,7 @@ func TestExpansionsPluginWExecution(t *testing.T) {
 			cmd := &update{YamlFile: "foo"}
 			So(cmd.Execute(ctx, comm, logger, conf), ShouldNotBeNil)
 			So(cmd.YamlFile, ShouldEqual, "foo")
-			So(conf.DynamicExpansions, ShouldResemble, &util.Expansions{})
+			So(conf.DynamicExpansions, ShouldResemble, util.Expansions{})
```

Comment:
nit: `So(conf.DynamicExpansions, ShouldBeEmpty)`

Reply:
I can't comment on unchanged lines, but if you assert it's empty here, we should also check in the test starting on L69 that the DynamicExpansions are set as expected by `expansions.update`.

Reply:
I removed this addition to this test altogether because the test is testing something else, there aren't actually any updates.